composer.json

```diff
@@ -87,7 +87,6 @@
     },
     "archive": {
         "exclude": [
-            "*",
```

Comment:
The trick is to first exclude all files and only add the files needed. That way we get a clean tarball with only the required files and no testing debugging or other files on production we don't want. That way the tarball is as lightweight as possible. Probably the best thing to do is  to add `!/component_info` and revert this change.

Reply:
Thank you for your suggestion. I've made the suggested updates.